native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -83,40 +78,34 @@ const styles = StyleSheet.create({
 
 const BpkCard = (props) => {
   const {
-    padded, children, focused, onPress, style, ...rest
+    padded,
+    children,
+    focused,
+    style: userStyle,
+    ...rest
   } = props;
 
-  const cardBaseStyle = [styles.card, styles.common];
-
-  if (padded) {
-    cardBaseStyle.push(styles.padded);
-  }
-
-  if (focused) {
-    cardBaseStyle.push(styles.focused);
-  }
+  const style = [styles.card];
```

Comment:
renamed `cardBaseStyle` to `style` - we are in the Card component we don;t need to prefix local variable with `card`.